Tests/ProjectDescriptionTests/EnvironmentTests.swift

```diff
@@ -63,4 +63,35 @@ final class EnvironmentTests: XCTestCase {
         let value = Environment.value(for: "1", environment: environment)
         XCTAssertNil(value)
     }
+
+    func testValueChecksForTuistPrefixedValuesFirst() {
+        let environment: [String: String] = [
+            "TUIST_NAME_SUFFIX": "0",
+            "NAME_SUFFIX": "1",
+        ]
+
+        // mimicing the camel cased dynamic member format
+        let value = Environment.value(for: "nameSuffix", environment: environment)
+        switch value {
+        case .boolean(false):
+            break
+        default:
+            XCTFail("Unexpected value. Got: \(String(describing: value)), expected: .boolean(false)")
+        }
```

Comment:
We can make Environment.Value equatable, so this becomes simply
```suggestion
        XCTAssertEqual(value, .boolean(false))
```

Reply:
Good call, updated